releases.json

```diff
@@ -389,6 +389,13 @@
       "1.9-1"
     ]
   },
+  "httplib": {
```

Comment:
Shouldn't it be "cpp-httplib" instead?

Reply:
I named it this way because the CMake target is called just `httplib`, but in https://github.com/yhirose/cpp-httplib/pull/470#issuecomment-630562373 I saw that the target has been named by a third party contributor... I asked the upstream owner, and I'm waiting for a response.